src/sentry/tasks/post_process.py

```diff
@@ -448,6 +448,10 @@ def run_post_process_job(job: PostProcessJob):
         try:
             pipeline_step(job)
         except Exception:
+            metrics.incr(
+                "sentry.tasks.post_process.post_process_group.exception",
+                tags={"issue_category": group_event.group.issue_category},
```

Comment:
`issue_category` is an enum. You'll want to convert this to either the int value, or maybe better, use the lower case name of the specific enum type.